src/util.ts

```diff
@@ -144,7 +149,16 @@ export const parseAppMeta = async <Platform extends SupportedPlatform>(
                 const baseApkFileName = manifestJson.split_apks?.find((apk) => apk.id === 'base');
                 const baseApkPath = baseApkFileName && (await writeFileFromZipToTmp(xapk, baseApkFileName.file));
                 await xapk.close();
-                return baseApkPath ? parseApk(baseApkPath) : undefined;
+
+                if (baseApkPath) {
+                    const meta = await parseApk(baseApkPath);
+                    if (meta)
+                        return {
+                            ...meta,
+                            md5: await hashFile(appPath, { algorithm: 'md5' }),
```

Comment:
I know this makes less sense idomatically, but why did you copy that line everywhere instead of just including it in `parseApk()`?

Reply:
Because that would change the behaviour, cf. the docstring:

> In the case of split APKs on Android, this will be the hash of the main APK. In the case of custom APK bundle formats (`.xapk`, `.apkm` and `.apks`), this will be the hash of the entire bundle.

Reply:
And while I personally don't see much use for the hash anyway, having the hash of the "outer" bundle format at least appears more useful to me.

Reply:
Ah, I see. Annoying, but correct then.